src/managers/KeybindManager.cpp

```diff
@@ -519,9 +520,13 @@ bool CKeybindManager::handleVT(xkb_keysym_t keysym) {
         // vtnr is bugged for some reason.
         unsigned int ttynum = 0;
 #if defined(VT_GETSTATE)
+        int            fd;
         struct vt_stat st;
-        if (!ioctl(0, VT_GETSTATE, &st))
-            ttynum = st.v_active;
+        if ((fd = open("/dev/tty", O_RDONLY | O_NOCTTY)) >= 0) {
```

Comment:
Can you apply to both `VT_GETSTATE` and `VT_GETACTIVE`?

<details><summary>example diff</summary>

```diff
diff --git a/src/managers/KeybindManager.cpp b/src/managers/KeybindManager.cpp
index d676001cd841..2a44137e74f8 100644
--- a/src/managers/KeybindManager.cpp
+++ b/src/managers/KeybindManager.cpp
@@ -519,19 +519,19 @@ bool CKeybindManager::handleVT(xkb_keysym_t keysym) {
 
         // vtnr is bugged for some reason.
         unsigned int ttynum = 0;
-#if defined(VT_GETSTATE)
-        int            fd;
-        struct vt_stat st;
+        int          fd;
         if ((fd = open("/dev/tty", O_RDONLY | O_NOCTTY)) >= 0) {
+#if defined(VT_GETSTATE)
+            struct vt_stat st;
             if (!ioctl(fd, VT_GETSTATE, &st))
                 ttynum = st.v_active;
-            close(fd);
-        }
 #elif defined(VT_GETACTIVE)
-        int vt;
-        if (!ioctl(0, VT_GETACTIVE, &vt))
-            ttynum = vt;
+            int vt;
+            if (!ioctl(fd, VT_GETACTIVE, &vt))
+                ttynum = vt;
 #endif
+            close(fd);
+        }
 
         if (ttynum == TTY)
             return true;
```
</details>

Reply:
done.